lib/services/databaseserver.go

```diff
@@ -267,6 +276,16 @@ func (s *DatabaseServerV2) CheckAndSetDefaults() error {
 	return nil
 }
 
+// Copy returns a copy of this database server object.
```

Comment:
Is this supposed to make a deep clone of the value? Since it is not doing it due to multiple attributes being pointers, maps and slices.

Reply:
Yeah I know, it wasn't supposed to make a full deep copy - just a new equivalent object for use in heartbeats.